crest/Assets/Crest/Crest/Scripts/OceanRenderer.cs

```diff
@@ -300,6 +306,16 @@ public enum DefaultClippingState
 
         public static OceanRenderer Instance { get; private set; }
 
+        /// <summary>
+        /// Is runtime environment without graphics card
+        /// </summary>
+        public static bool RunningWithoutGPU => SystemInfo.graphicsDeviceID == 0 || (Instance != null ? Instance._forceNoGPU : false);
```

Comment:
*SystemInfo.graphicsDeviceID* is zero when there is only integrated graphics (tested on MacBook). There is [UnityEngine.Rendering.GraphicsDeviceType](https://docs.unity3d.com/2019.4/Documentation/ScriptReference/Rendering.GraphicsDeviceType.html) which works.

Reply:
damn the internet for being wrong. made a second attempt and it seems to be good now, thanks